lib/jekyll/commands/help.rb

```diff
@@ -12,9 +12,9 @@ def init_with_program(prog)
             c.action do |args, _|
               cmd = (args.first || "").to_sym
               if args.empty?
-                puts prog
+                Jekyll.logger.info prog
               elsif prog.has_command? cmd
-                puts prog.commands[cmd]
+                Jekyll.logger.info prog.commands[cmd]
```

Comment:
We might have to call `to_s` on `prog.commands[cmd]` here.

Reply:
done